package.json

```diff
@@ -30,6 +31,8 @@
     "lint": "standard | snazzy",
     "precommit": "npm run lint -s",
     "pretest": "npm run lint -s",
-    "test": "NODE_ENV=test ava"
+    "test": "NODE_ENV=test ava",
+    "build": "spike compile -e production",
+    "watch": "spike watch"
```

Comment:
It is no longer necessary to have spike installed globally to run your project, you can just use `npm run watch` to ensure your spike version matches that of the project. With the latest version of the global spike CLI, it will look in your project and use a local spike install if present as well, so that its easy to have multiple spike projects on different versions.